addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3497,6 +3497,15 @@ var SnippetsMenu = Widget.extend({
                     dynamicSvg.src = colorCustomizedURL.pathname + colorCustomizedURL.search;
                 });
 
+                // The iframes must be loaded when the drag-and-drop starts and not when the snippet
+                // menu is loaded.
+                const iframeEl = $toInsert[0].querySelector("iframe[data-src]");
+                if (iframeEl) {
+                    const iframeDataSrc = iframeEl.getAttribute("data-src");
+                    iframeEl.removeAttribute("data-src");
+                    iframeEl.setAttribute("src", iframeDataSrc);
```

Comment:
Less line + use the `dataset`

```suggestion
                    iframeEl.setAttribute("src", iframeEl.dataset.oSrcOnDrop);
                    delete iframeEl.dataset.oSrcOnDrop
```